app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -952,8 +960,9 @@ class AdvancedFilter extends React.Component {
             <option value="greater-than-equal">greater than or equal to</option>
             <option value="greater-than">greater than</option>
```

Comment:
`changeFilterNumberOption` should be update to set the value to null when blank is selected